src/ArcxAnalyticsSdk.ts

```diff
@@ -391,6 +391,16 @@ export class ArcxAnalyticsSdk {
   /** Initialises the Analytics SDK with desired configuration. */
   static async init(apiKey: string, config?: Partial<SdkConfig>): Promise<ArcxAnalyticsSdk> {
     const sdkConfig = { ...DEFAULT_SDK_CONFIG, ...config }
+    /*
+      TODO: Remove this if statement aproximately in February.
+      Clean local storage in case if users with previous versions saved wrong indentity_id when ARCX_ANALYTIC_API_KEY was incorrect
```

Comment:
Why Feb? Also I would add the year to this (e.g. Feb 2023)

Also, if its a TODO, I would add a github issue and make it priority so we don't forget.

Reply:
https://github.com/arcxmoney/analytics-sdk/issues/96